packages/roosterjs-editor-api/lib/table/insertTable.ts

```diff
@@ -46,7 +44,7 @@ export default function insertTable(
             let vtable = new VTable(table);
             vtable.applyFormat(format);
             vtable.writeBack();
-            editor.insertNode(fragment);
+            editor.insertNode(table);

```

Comment:
now with this change, the insert table behavior is really regressed.

Repro:
1. insert a 3*4 table
    
![image](https://user-images.githubusercontent.com/23065085/193155464-fc41b0fa-d149-4944-a2ca-d99b41e9ae1c.png)
2. Go to row 3, colomn 1 (the first cell of the last row)
4. insert another 3*4 table

Original behavior (expected):
The 3*4 table is expanded to be a 6*4 table
![image](https://user-images.githubusercontent.com/23065085/193155494-5f6e3d8a-7995-4f0d-9aab-9dc991c89744.png)


New behavior:
I got a 3*4 table nested inside the cell

![image](https://user-images.githubusercontent.com/23065085/193155516-3574abe8-e26d-47a0-a399-295e61ba0448.png)

Reply:
Discussed offline and we can accept this new behavior.